apps/ai-image-tagging/lambda/usage.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
+
+const AWS = require('aws-sdk');
+
+const documentClient = new AWS.DynamoDB.DocumentClient();
+
+module.exports = async spaceId => {
+  const now = new Date();
+  const firstDay = new Date(now.getFullYear(), now.getMonth(), 1);
```

Comment:
```suggestion
  const firstDay = Date.UTC(now.getFullYear(), now.getMonth(), 1);
```

It pays to be really explicit with timezones. This will also directly give you msecs.